release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1309,6 +1315,12 @@ In addition, the following commands related to the format have been removed from
 
 This release removes the Prometheus Adapter, which was a Technology Preview feature.
 
+[id="ocp-4-8-oauth-secure-token-annotation"]
+==== Secure token storage annotation recognition is removed 
+
+The `authentication` and `openshift-apiserver` Operators now ignore the `oauth-apiserver.openshift.io/secure-token-storage` annotation when picking the audit policy of a cluster. Audit policies now use `secure-` by default. For more information, see link:https://bugzilla.redhat.comshow_bug.cgi?id=1879182[*BZ#1879182*].
```

Comment:
This link is broken. Try: 

link:https://bugzilla.redhat.com/show_bug.cgi?id=1879182[*BZ#1879182*].